pyproject.toml

```diff
@@ -0,0 +1,115 @@
+[tool.poetry]
+name = "aries_cloudagent"
+version = "0.10.0-rc0"
+description = ""
+authors = ["Your Name <you@example.com>"]
+license = "Apache-2.0"
+readme = "README.md"
+packages = [{include = "aries_cloudagent"}]
+classifiers = [
+            "Programming Language :: Python :: 3",
+            "License :: OSI Approved :: Apache Software License",
+	    "Operating System :: OS Independent",
+]
+repository = "https://github.com/hyperledger/aries-cloudagent-python"
+
+
+[tool.poetry.dependencies]
+python = "^3.9"
+aiohttp="~3.8.1"
+aiohttp-apispec="~2.2.1"
+aiohttp-cors="~0.7.0"
+aioredis="~2.0.0"
+apispec="~3.3.0"
+async-timeout="~4.0.2"
+base58="~2.1.0"
+ConfigArgParse="~1.5.3"
+deepmerge="~0.3.0"
+ecdsa="~0.16.1"
+jsonpath_ng="1.5.2"
+Markdown="~3.1.1"
+markupsafe="2.0.1"
+marshmallow="~3.20.1"
+msgpack="~1.0"
+multiformats="~0.2.1"
+nest_asyncio="~1.5.5"
+packaging="~23.1"
+portalocker="~2.7.0"
+prompt_toolkit=">=2.0.9,<2.1.0"
+pydid="~0.3.6"
+pyjwt="~2.8.0"
+pyld="~2.0.3"
+pynacl="~1.5.0"
+python-dateutil="~2.8.1"
+python-json-logger="~2.0.7"
+pytz="~2021.1"
+pyyaml="~6.0.1"
+qrcode = {version = ">=6.1,<7.0", extras = ["pil"]}
+requests="~2.31.0"
+rlp="1.2.0"
+unflatten="~0.1"
+asyncpg = ">=0.25.0,<0.26.0"
+web-py = ">=0.62,<1.0"
+pygments = ">=2.10,<3.0"
+
+# [tool.poetry.group.test.dependencies]
+asynctest= { version = "0.13.0", optional = true }
+pytest= { version = "~7.4.0", optional = true }
+pytest-asyncio= { version = "0.14.0", optional = true }
+pytest-cov= { version = "2.10.1", optional = true }
+pytest-flake8= { version = "1.1.1", optional = true }
+mock= { version = "~4.0", optional = true }
```

Comment:
These should still be in the dev dependency group. They're used only in development so should be treated as a dependency group rather than an extra.

Reply:
That makes perfect sense I have just moved them into the dev.dependancy group. Thanks for the feedback